sensor/scanner/openvas/openvas.go

```diff
@@ -32,8 +32,9 @@ import (
 // responsible for handling processes of openvas. It also is able to start and
 // stop scans via openvas.
 type OpenVASScanner struct {
-	procs map[string]*os.Process // Process List for running scans
-	mutex *sync.Mutex            // For thread save management of processes
+	procs         map[string]*os.Process // Process List for running scans
+	mutex         *sync.Mutex            // For thread save management of processes
```

Comment:
You can also do:
```
type OpenVASScanner struct {
   sync.RWMutex
   ...
}
```
then you don't have to writer mutex.Lock, mutex.Unlock but can do it directly on the struct.